internal/analytics/client/sync/reporters/pixel.go

```diff
@@ -14,7 +16,13 @@ type PixelReporter struct {
 }
 
 func NewPixelReporter() *PixelReporter {
-	return &PixelReporter{"https://state-tool.s3.amazonaws.com/pixel"}
+	var pixelUrl string
+
+	// Attempt to get the value for the pixel URL from the environment.  Fall back to default if that fails
+	if pixelUrl = os.Getenv(constants.AnalyticsPixelOverrideEnv); pixelUrl == "" {
+		pixelUrl = constants.DefaultAnalyticsPixel
+	}
+	return &PixelReporter{pixelUrl}
```

Comment:
What is the use-case of this? I think it might already be addressed:

https://github.com/ActiveState/cli/blob/ec562261f7b30dcebad79f30ab327f217090ebdc/internal/constants/constants.go#L124

Reply:
That env var allows us to override which pixel we send data to during testing, the one you referenced allows you to use a TestReporter instead of a PixelReporter which doesn't use a pixel at all.

Reply:
I understand that, I'm saying we have a way of debugging whether analytics are firing already. I'd prefer not to have multiple ways of doing the same thing.

I don't think it's your job to test if the pixel mechanic specifically works. What you care about is that the events are triggered. Whether pixels work ought to be out of scope for what you're working on.

Reply:
DW specifically asked for us to be able to redirect events to a secondary pixel for them to test processing the data at their end without interfering with data sent to the production pixel.  This addresses that use case.